test/integration/services/Common.go

```diff
@@ -0,0 +1,42 @@
+package services
```

Comment:
The naming style used for filenames in this project (and most Go projects) is all lowercase with underscore separation when necessary, i think it's better to name this file `common.go`